README.md

```diff
@@ -1 +1,228 @@
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Install_Ruby_on_Rails_framework]**
+- **[Use_Ruby_Gems_as_a_software_packages_system]**
+- **[Use_database_migration_files_to_maintain_database_schema]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo coming soon...](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+  ```sh
+     gem install rails
+  ```
+### Setup
+
+Clone this repository to your desired folder:
+
+  ```sh
+    cd my-folder
+    git clone https://github.com/Goruchie/blog-app.git
+  ```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd blog-app
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@Goruchie](https://github.com/Goruchie)
+- Twitter: [@twitterhandle](https://twitter.com/goruchie2)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/marcosgualtero)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[Validations_and_Model_specs]**
+- [ ] **[Processing_data_in_model]**
+- [ ] **[Setup_and_controllers]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Goruchie/blog-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank @luccamicale for sharing me his knowledge.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT) licensed.
+
```

Comment:
- [ ] I think it would be better if you fix the broken `[MIT](./MIT)` link in your README file to make it more professional as it shows a **404** error page once clicked. This is happening because of the typo error you have. It should be `[MIT](./MIT.md)` instead of `[MIT](./MIT)` as your license file is named `MIT.md`.